cs3/app/registry/v1beta1/registry_api.proto

```diff
@@ -120,6 +122,24 @@ message ListAppProvidersResponse {
   repeated ProviderInfo providers = 3;
 }
 
+message ListSupportedMimeTypesRequest {
+  // OPTIONAL.
+  // Opaque information.
+  cs3.types.v1beta1.Opaque opaque = 1;
+}
+
+message ListSupportedMimeTypesResponse {
```

Comment:
at least for the Web frontend we would need the app_name in here because this does not equal the provider_name!?

Reply:
Yep, I'll correct the naming. It actually stores the name of the app only